content/contact.md

```diff
@@ -3,11 +3,27 @@ description: Find out the different ways to contact Pension Wise and how to make
 ---
 # Contact us
 
-You can email Pension Wise at <contact@pensionwise.gov.uk>. We'll soon have an address for you to write to us by post.
+^Pension Wise can’t answer specific questions about your pension.^
 
-We can't answer specific questions about your pension and don’t recommend products or tell you what to do with your money.
+For independent help contact the: 
 
-If you’d like to speak to a guidance specialist about your pension options, you can [book an appointment](/appointments).
+- [Pensions Advisory Service](http://www.pensionsadvisoryservice.org.uk/) 
```

Comment:
Shouldn't they be "The Pensions Advisory Service" not "Pensions Advisory Service"?